CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 0.17.2 (unreleased)
 ===================
 
+ami
+---
+
+- Update code and unit tests to use new ami analyze algorithm [#5377, 5312, 5323]
```

Comment:
The PR that merges all the new code into master is #5390 , so this should be updated to remove the references to 5377, 5312, and 5323 and just reference 5390.